### DISCUSSION TITLE
Is there any way to count how much and show where users have cloned or downloaded my code from?

### DISCUSSION THREAD

**Discussion by PrideLab** [community/community#40154] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Our project on Github is part of a scientific research project, and we would like to know how many people have cloned or downloaded our code on Github, and roughly where they came from, so that we can use it as proof of our results declaration. Do GitHub has such a feature, or are there any plans to add it? 🤔

**Comment by moehmeni** [comment]
To see the numbers, navigate to the main page of the repo, and go to Insights/Traffic, but the location of visitors/cloners is not visible.

**Comment by jfullstackdev** [comment, marked as answer]
one, you have the option to install a simple page counter on your repo, 

https://github.com/jdevstatic/page-metrics-tutorial

two, use the default of GitHub at the Traffic tab

three, if your repo has releases you can use this

https://tooomm.github.io/github-release-stats/
